db/sqlite-schema.ts

```diff
@@ -0,0 +1,133 @@
+import {
```

Comment:
Delete this file and don't introduce a second schema. Delete the generated migration files too